ctms/ingest_stripe.py

```diff
@@ -327,6 +329,25 @@ def ingest_stripe_price(
     }
 
 
+def _delete_orphaned_line_items(
+    db_session: Session, invoice: StripeInvoice, line_item_data_ids: Set[str]
+) -> Set[str]:
+    """Remove any orphaned invoice items, returning IDs of the deleted items
+
+    Orphaned line items are line items that exist on the invoice that don't
+    exist in the incoming line item data.
+    """
+    stmt = (
+        delete(StripeInvoiceLineItem)
+        .where(StripeInvoiceLineItem.stripe_invoice_id == invoice.stripe_id)
+        .where(StripeInvoiceLineItem.stripe_id.notin_(line_item_data_ids))
+        .execution_options(synchronize_session=False)
```

Comment:
Can you add a comment specifying that we're not interested in synchronizing objects here?